README.rst

```diff
@@ -1,6 +1,6 @@
-****************
+************************
 django CMS FIL Bootstrap
-****************
+************************
 
 ============
 Installation
```

Comment:
I know it's probably obvious to you but to someone who doesn't understand the purpose of bootstrap it would be really helpful to include a `Why use this?` section, which provides some background and an `architecture` section, which explains in broad terms how the add-on works.

We're going to have to get used to doing this sort of thing.